src/main/java/com/sprints/SprintsApiApplication.java

```diff
@@ -3,12 +3,27 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 
+import com.sprints.interceptors.UndeclaredParamsHandlerInterceptor;
+
+@SuppressWarnings("deprecation")
```

Comment:
Why we need suppress deprecate warnings ? We should not use deprecate implementations

Reply:
Fixed